manual/tdr-reporting.md

```diff
@@ -0,0 +1,45 @@
+# TDR Reporting
+
+These are instructions for running the reports TDR provide via our GitHub action.
+
+## Requesting access for a new user
+You will need a GitHub account. If you don't already have one, [you can create one](https://github.com/signup). You will need to add two-factor authentication to your account. GitHub have [a guide to setting this up](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication)
+
+If you are already a member of the `nationalarchives` GitHub organisation then you will already have access.
+
+If not, ask the TDR team to add you to the `tdr-reporting` repository as a collaborator. You can contact us [on our Slack channel](https://tna-digital.slack.com/archives/CJYJYG54L) Once this is done, we will let you know.
+
+## Run the reports
+Go to the [tdr-reporting actions page](https://github.com/nationalarchives/tdr-reporting/actions/workflows/run.yml). You will see a page similar to this
+
+![](./images/reporting_run_action.png)
+ 
+Click "Run workflow" which will give you options like this:
+
+![](./images/reporting_drop_down.png)
+You can ignore the first dropdown with Branch: main in it. The other three can be configured:
```

Comment:
Think a line break is needed after the image link. When I view the page, the test is appearing on the same line as the bottom of the image.

Reply:
I've put the line break in, cheers.